CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.5)
 
-project(CATChem VERSION 0.1.0 LANGUAGES Fortran)
+project(CATChem VERSION 0.1.0 LANGUAGES Fortran C CXX)
```

Comment:
Without adding these, was getting this error:
```
CMake Error in src/process/chem/CMakeLists.txt:
  No known features for CXX compiler

  ""

  version .
```

Reply:
And it seems like really it's only necessary to add CXX

Reply:
@K20shores do you know if there's a way to get MUSICA to build without changing this, by specifying something elsewhere? Just curious.

Reply:
@zmoon I think this is required. If not, I don't think the linker will be properly configured. I could be wrong though

Reply:
@zmoon I was able to build your fork

Reply:
Ah, if you want to only enable this langauges when musica is built, somewhere you can do something like this

```
if (ENABLE_MUSICA)
    enable_language(C)
    enable_language(CXX)
endif()
```